### DISCUSSION TITLE
Update Postal server to allow passwords on smtp

### DISCUSSION THREAD

**Discussion by engrpeters** [postalserver/postal#2317] [Feature suggestions]
You all need to update postal server to allow password smtp_relays . It is very important
My PostalServer wouldn't allow me receive mails, I can only send . I can't use Sendgrid or Gmail's SMTP either because they require authentication .

**Comment by willpower232** [comment]
Someone commented recently with the change they made to Postal to support this so you should be able to apply the same change via a docker-compose volume or making your own docker image.

https://github.com/postalserver/postal/issues/48#issuecomment-1065875689
